source/assets/js/classes/fef-dom-observer.js

```diff
@@ -1,12 +1,50 @@
 $(document).ready(function () {
     let observer = new FefDomObserver();
-    observer.triggerDomChangedEvent();
+    observer.triggerDomInitEvent();
 });
 
+export const DOM_INIT_EVENT = 'fef.dom.initialized';
 export const DOM_CHANGED_EVENT = 'fef.dom.changed';
+export const DOM_MUTATION = 'fef.dom.initialized fef.dom.changed';
 
 export class FefDomObserver {
 
+    constructor () {
+        const targetNode = document.querySelector('body');
+        // Options for the observer (which mutations to observe)
+        const config = { attributes: false, childList: true, subtree: false };
+
+        // Callback function to execute when mutations are observed
+        const callback = (mutationsList, observer) => {
+            let mutationsOfInterest = 0;
+
+            for(let mutation of mutationsList) {
+                if (mutation.type === 'childList' && mutation.addedNodes.length > 0) {
+                    mutationsOfInterest++;
+                }
+            }
+
+            if (mutationsOfInterest > 0) {
```

Comment:
Du willst ja nur wissen, ob _überhaupt_ eine Mutation relevant ist, nicht _wieviele_:
```
if (mutationsList.some(mutation => mutation.type === 'childList' && mutation.addedNodes.length > 0)) {
  ...
}
```

Reply:
In der console wollte ich es ursprünglich schon wissen wieviele ;)
Aber ja, so siehts einfacher aus, merci

Reply:
Dann wär ein `.filter` dein Kolleg 😄